### PR TITLE
Implement json error pages

### PR DESCRIPTION
**Pull request by davispuh**
Implement json error pages.
This is useful when JSON backend/API doesn't have nice error info and might return blank page.

To use with Nginx:

```nginx
map $http_accept $errorExtension
{
    default                    html;
    ~application/json          json;
    ~application/activity+json json;
}


error_page 400 /ErrorPages/HTTP400.$errorExtension;
error_page 401 /ErrorPages/HTTP401.$errorExtension;
error_page 403 /ErrorPages/HTTP403.$errorExtension;
# ...
```

This will cause that when client has `Accept: application/json` header then it will get JSON response.

### CODE REVIEW

**Comment by davispuh** [comment]
Ping? Could you merge my PRs?

**Comment by davispuh** [comment]
What about this one?

**Comment by AndiDittrich** [comment]
tldr: technically out of scope

**Comment by davispuh** [comment]
How so? I need a solution to handle all error pages even for JSON APIs and this looks like best tool. It doesn't make sense to have separate tool just for JSON when this could do it all.
